.travis.yml

```diff
@@ -1,4 +1,6 @@
-language: bash
+language: go
+go:
+  - "1.12"
```

Comment:
does this work the same as 1.12.x?

Reply:
I believe so: https://docs.travis-ci.com/user/languages/go/#specifying-a-go-version-to-use

Reply:
The quotes are important because `1.10` was interpreted as `1.1` because YAML